exercise_03.py

```diff
@@ -0,0 +1,20 @@
+def foot_to_meters(foot):
+    meter = foot * 0.305
+    return "{0:.3f}".format(meter)
+
+
+def meters_to_foot(meter):
+    foot = meter / 0.305
+    return "{0:.3f}".format(foot)
+
+
+def main():  
+    print(f"{'Feet':<10}{'Meter':<10}{'|':<10}{'Meter':<10}{'Feet':<10}")
```

Comment:
Can we create a function called `print_table` in order to do exactly as the name indicates? I do not like to see too much low level code in the `main` function.